uber/templates/registration/new_base.html

```diff
@@ -185,7 +260,6 @@
             <form method="post" id="new_checkin_{{ attendee.id }}" action="new_checkin">
```

Comment:
Aha, I actually wondered about this. I have time today and I want to see about using jquery forms for our ajax (and ajaxify the checkin button) so I'll do this as well.